docs/07-my-first-awesome.md

```diff
@@ -171,7 +171,28 @@ A new window will appear and a new awesome instance runs inside of it.
 
 ## Add widgets
 
-TODO: Write this section.
+Before the following line:
+
+    mylauncher = awful.widget.launcher({ image = beautiful.awesome_icon,
+
+Add this to create a new widget:
+
+    praisewidget = wibox.widget.textbox()
+    praisewidget:set_text("You are great!")
```

Comment:
Could it use `.text`? The issue I see is that `:set_text()` is not explicitly documented and this could cause some confusion to new users (the intended audience of this article)